problems/arraysWithLoops.js

```diff
@@ -66,15 +121,44 @@ function findSmallest() {}
  * @returns {number} second smallest value
  */
 
-function findSecondSmallest() {}
+function findSecondSmallest(nums) {
+
+  let smallest = Infinity;
+  let second = Infinity;
+  for (let i = 0; i < nums.length; i++) {
+    const num = nums[i];
+    if (num < smallest) {
+      second = smallest;
+      smallest = num;
+    } else if (num < second) {
+      second = num;
+    }
+  }
+  return second;
+}
 
 /**
  * Takes in array and returns the second largest number in the array
  * @param {number[]} nums
  * @returns {number} second largest value
  */
 
-function findSecondLargest() {}
+function findSecondLargest(nums) {
+
+  let largest = Infinity;
+  let secondLargest = Infinity;
```

Comment:
If we start at Infinity and are looking for the largest number, Infinity will always win. What's the smallest number in the world that we can start with?